datasets/ronec/ronec.py

```diff
@@ -59,53 +57,52 @@ def __init__(self, **kwargs):
 class RONEC(datasets.GeneratorBasedBuilder):
     """RONEC dataset"""
 
-    VERSION = datasets.Version("1.0.0")
+    VERSION = datasets.Version("2.0.0")
     BUILDER_CONFIGS = [
         RONECConfig(name="ronec", version=VERSION, description="RONEC dataset"),
     ]
```

Comment:
It would be nice to still let users access the V1 for reproducibility of all the work before RONEC v2

Do you think you could add a configuration that allows users to load RONEC V1 ? You can use the new links to the V1 data files